test/data/manifests/fedora_34-x86_64-oci-boot.json

```diff
@@ -9514,15 +12161,18 @@
       "dracut-config-generic-055-6.fc34.x86_64",
       "e2fsprogs-1.45.6-5.fc34.x86_64",
       "e2fsprogs-libs-1.45.6-5.fc34.x86_64",
+      "efi-filesystem-5-4.fc34.noarch",
+      "efibootmgr-16-10.fc34.x86_64",
+      "efivar-libs-37-15.fc34.x86_64",
       "elfutils-debuginfod-client-0.186-1.fc34.x86_64",
       "elfutils-default-yama-scope-0.186-1.fc34.noarch",
       "elfutils-libelf-0.186-1.fc34.x86_64",
       "elfutils-libs-0.186-1.fc34.x86_64",
       "expat-2.4.1-1.fc34.x86_64",
       "fedora-gpg-keys-34-2.noarch",
-      "fedora-release-cloud-34-39.noarch",
+      "fedora-release-34-39.noarch",
       "fedora-release-common-34-39.noarch",
-      "fedora-release-identity-cloud-34-39.noarch",
```

Comment:
This image-info report is apparently outdated and does not correspond with the manifest. There is `fedora-release-identity-cloud-34-39.noarch.rpm` in the manifest sources, but the image-info report does not contain it.

Reply:
Will regenerate all manifests for f34 and f35 with aarch64 and x86_64. I'll pay attention on this manifest in particular.

Reply:
Fixed. Please review.